food-for-thought/app/home.tsx

```diff
@@ -56,7 +56,7 @@ const Home = () => {
     <View style={styles.container}>
       <Header homepage={true}></Header>
       {/* Card for the Restaurant finder */}
-      <TouchableOpacity onPress={() => router.push('/map')}>
+      <TouchableOpacity onPress={() => router.push('/warning')}>
```

Comment:
This works for me! Just wondering whether we want to actually be routing to a new page for this or if we want the warning to be a component that's rendered as some kind of a pop-up modal on the Home Page? Cause you can see the swipe when you click on the Card🤔